### PR TITLE
[ADF-3108] Search - facetQueries panel renders when not defined in configuration

### PR DESCRIPTION
**Pull request by pionnegru**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

https://issues.alfresco.com/jira/browse/ADF-3108

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/content-services/search/components/search-filter/search-filter.component.ts [hunk 1]
```diff
@@ -71,6 +71,10 @@ export class SearchFilterComponent implements OnInit {
         }
     }
 
+    get isFacetQueriesDefined() {
+        return !!(this.queryBuilder.config && this.queryBuilder.config.facetQueries);
```
Comment (DenisVuyka) [inline lib/content-services/search/components/search-filter/search-filter.component.ts]: They still can be defined as `[]`, so I suggest checking the length as well

Comment (pionnegru) [reply lib/content-services/search/components/search-filter/search-filter.component.ts]: should we make this accessible?

Comment (DenisVuyka) [reply lib/content-services/search/components/search-filter/search-filter.component.ts]: Yes, I'm fine with making that public

**Comment by pionnegru** [comment]
@DenisVuyka updated

**Pull request merged**